src/features/query-bar/components/filter/Condition.js

```diff
@@ -169,17 +174,48 @@ const Condition = ({
     };
 
     const renderPrefixField = () => {
+        const { prefix } = condition;
+        let message = '';
+        switch (prefix) {
+            case WHERE:
+                message = { ...messages.prefixLabelWhereText };
+                break;
+            case AND:
+                message = { ...messages.prefixButtonAndText };
+                break;
+            case OR:
+                message = { ...messages.prefixButtonOrText };
+                break;
+            default:
+                break;
+        }
+
+        const prefixOptions = getFormattedOptions(
+            [AND, OR].map(key => ({
+                displayName: key,
+                value: key,
+            })),
+        );
+
         return (
             <div className="filter-item-prefix-container">
-                <p className="filter-item-prefix-text">
-                    <FormattedMessage {...messages.prefixButtonText} />
-                </p>
+                {prefix === WHERE ? (
+                    <p className="filter-item-prefix-text">{formatMessage(message)}</p>
```

Comment:
We should use `<FormattedMessage>` wherever possible.

Reply:
Fixed